packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -1075,3 +1080,63 @@ export function unmountFundamentalComponent(
 export function getInstanceFromNode(node: HTMLElement): null | Object {
   return getClosestInstanceFromNode(node) || null;
 }
+
+export function mountEventListener(listener: ReactDOMListener): void {
+  if (enableUseEventAPI) {
+    const {target} = listener;
+    if (target === window) {
+      // TODO
+    } else {
+      attachElementListener(listener);
+    }
+  }
+}
+
+export function unmountEventListener(listener: ReactDOMListener): void {
+  if (enableUseEventAPI) {
+    const {target} = listener;
+    if (target === window) {
+      // TODO
+    } else {
+      detachElementListener(listener);
+    }
+  }
+}
+
+export function validateEventListenerInstance(
```

Comment:
Just curious why you've renamed all of the params to `target` but this method calls it "instance"